internal/postgres/config.go

```diff
@@ -313,27 +313,29 @@ manageAutogrowAnnotation() {
   local trigger=$2
   local maxGrow=$3
 
-  size=$(df --block-size=M /"${volume}" | awk 'FNR == 2 {print $2}')
-  use=$(df /"${volume}" | awk 'FNR == 2 {print $5}')
+  size=$(df --block-size=M /"${volume}")
```

Comment:
🔧 Since this is in a different package, I think it would be worth updating the comments to explain what the script does here. I think it should be more or less a copy/paste though.
See https://github.com/CrunchyData/postgres-operator/blob/main/internal/pgbackrest/config.go#L640-L649